repos/fdbt-site/src/pages/multipleOperatorsServiceList.tsx

```diff
@@ -46,12 +45,13 @@ const MultipleOperatorsServiceList = ({
     dataSourceAttribute,
 }: MultipleOperatorsServiceListProps): ReactElement => (
     <FullColumnLayout title={pageTitle} description={pageDescription}>
-        <SwitchDataSource
+        {/* removed as TNDS is being disabled until further notice */}
+        {/* <SwitchDataSource
             dataSourceAttribute={dataSourceAttribute}
             pageUrl="/multipleOperatorsServiceList"
             attributeVersion="multiOperator"
             csrfToken={csrfToken}
-        />
+        /> */}
         <CsrfForm action="/api/multipleOperatorsServiceList" method="post" csrfToken={csrfToken}>
             <>
                 <ErrorSummary errors={errors} />
```

Comment:
getServerSideProps has more changes that are needed, as it is still checking for the presence of TNDS things. See serviceList.tsx for examples .

Reply:
[bd4eab1](https://github.com/fares-data-build-tool/create-data/pull/572/commits/bd4eab107f7288cba4384b8884cf9d1b20382c45)